test/phx_tools_web/live/index_test.exs

```diff
@@ -51,12 +65,11 @@ defmodule PhxToolsWeb.PhxToolsLive.IndexTest do
     end
   end
 
-  describe "macOS instructions page" do
+  describe "/macOS" do
```

Comment:
Does mixing cases work (/macOS vs /macos)? We've had an issue with /Linux in the browsers. What about /macOS?

Reply:
About this, I think it is good for readability, plus it's been in place and has not caused any issues.
It only came up when we switched to /Linux
I also made this change you are suggesting already, I don't know why it's not reflecting here.